### PR TITLE
PP-4 : Form design of create project

### PR DESCRIPTION
**Pull request by sanjay-techuz**

### CODE REVIEW

File: src/components/Form/ExperimentalProject/CreateExperimentalProject.jsx [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+      value: 'FIX'
+    },
+    {
+      name: 'TM',
+      value: 'TM'
+    }
+  ];
+
+  const validationSchema = Yup.object().shape({
+    project_name: Yup.string().required('Project name is Required'),
```
Comment (ketanp-techuz) [inline src/components/Form/ExperimentalProject/CreateExperimentalProject.jsx:68]: Required shuold be the same as below line msg

File: src/components/Form/Project/CreateProject.jsx [hunk 2]
```diff
@@ -59,123 +53,170 @@ export default function CreateProject() {
...
+    },
+    {
+      name: 'TM',
+      value: 'TM'
+    }
+  ];
 
-  const [members, setMembers] = useState([]);
+  const validationSchema = Yup.object().shape({
+    project_name: Yup.string().required('Project name is Required'),
```
Comment (ketanp-techuz) [inline src/components/Form/Project/CreateProject.jsx:68]: msg should be the same as below line

File: src/components/Form/ExperimentalProject/CreateExperimentalProject.jsx (hunk 1 above)
Comment (sanjay-techuz) [reply src/components/Form/ExperimentalProject/CreateExperimentalProject.jsx:68]: @ketanp-techuz means? can you please provide example here.

**Comment by ketanp-techuz** [comment]
Now onwards don't push the package.lock file.

**Pull request merged**